src/Transfer/common.js

```diff
@@ -0,0 +1,113 @@
+import './types.js'
+import { colors } from '../theme.js'
+
+export const borderColor = colors.grey400
+export const borderRadius = '3px'
+
+/**
+ * Click modes when clicking on an option with/without
+ * a modifier key (ctrl, alt, cmd, shift)
+ */
+
+// no or multiple modifier keys
+export const REPLACE_MODE = Symbol('REPLACE_MODE')
+// add/remove options from selection
+export const ADD_MODE = Symbol('ADD_MODE')
+// create selection range
+export const RANGE_MODE = Symbol('RANGE_MODE')
```

Comment:
We could just use regular string here right? I could see the use of a symbol if we have sth like a object where we don't want the key to be accidentally overwritten or something, but in this case I'd just go with a string const. But let me know if I'm missing something.